statsmodels/tools/decorators.py

```diff
@@ -2,7 +2,65 @@
 from statsmodels.tools.sm_exceptions import CacheWriteWarning
 import warnings
 
-__all__ = ['cache_readonly', 'cache_writable']
+__all__ = ['cache_readonly', 'cache_writable', 'deprecated_alias']
+
+
+def deprecated_alias(old_name, new_name, remove_version=None, msg=None,
```

Comment:
Is this vendored from somewhere? Might be goog to make it clear in the docstring if true, in case of changes.

Reply:
I implemented the original in sm2, then you made some improvements in your PR (which I may have subsequently updated the original with)